app/Resources/views/grid/content-simple.html.twig

```diff
@@ -6,6 +6,12 @@
 
         <div class="content-container grid__item content_alerts-container">
 
+            {% for infoBar in infoBars %}
```

Comment:
As this is repeated in multiple templates it would be good to have an include here. Not essential.

Reply:
Include added for infobar announcement block